packages/react-dom/src/events/DOMEventProperties.js

```diff
@@ -41,104 +122,108 @@ const eventPriorities = new Map();
-  DOMTopLevelEventTypes.TOP_DRAG_ENTER, 'dragEnter',
-  DOMTopLevelEventTypes.TOP_DRAG_EXIT, 'dragExit',
-  DOMTopLevelEventTypes.TOP_DRAG_LEAVE, 'dragLeave',
-  DOMTopLevelEventTypes.TOP_DRAG_OVER, 'dragOver',
-  DOMTopLevelEventTypes.TOP_MOUSE_MOVE, 'mouseMove',
-  DOMTopLevelEventTypes.TOP_MOUSE_OUT, 'mouseOut',
-  DOMTopLevelEventTypes.TOP_MOUSE_OVER, 'mouseOver',
-  DOMTopLevelEventTypes.TOP_POINTER_MOVE, 'pointerMove',
-  DOMTopLevelEventTypes.TOP_POINTER_OUT, 'pointerOut',
-  DOMTopLevelEventTypes.TOP_POINTER_OVER, 'pointerOver',
-  DOMTopLevelEventTypes.TOP_SCROLL, 'scroll',
-  DOMTopLevelEventTypes.TOP_TOGGLE, 'toggle',
-  DOMTopLevelEventTypes.TOP_TOUCH_MOVE, 'touchMove',
-  DOMTopLevelEventTypes.TOP_WHEEL, 'wheel',
+  'drag', 'drag',
+  'dragenter', 'dragEnter',
+  'dragexit', 'dragExit',
+  'dragleave', 'dragLeave',
+  'dragover', 'dragOver',
+  'mousemove', 'mouseMove',
+  'mouseout', 'mouseOut',
+  'mouseover', 'mouseOver',
+  'pointermove', 'pointerMove',
+  'pointerout', 'pointerOut',
+  'pointerover', 'pointerOver',
+  'scroll', 'scroll',
+  'toggle', 'toggle',
+  'touchmove', 'touchMove',
+  'wheel', 'wheel',
 ];
 
+export const ANIMATION_END = getVendorPrefixedEventName('animationend');
+export const ANIMATION_START = getVendorPrefixedEventName('animationstart');
+export const ANIMATION_ITERATION = getVendorPrefixedEventName(
+  'animationiteration',
+);
+export const TRANSITION_END = getVendorPrefixedEventName('transitionend');
+
 // prettier-ignore
 const continuousPairsForSimpleEventPlugin = [
-  DOMTopLevelEventTypes.TOP_ABORT, 'abort',
-  DOMTopLevelEventTypes.TOP_ANIMATION_END, 'animationEnd',
-  DOMTopLevelEventTypes.TOP_ANIMATION_ITERATION, 'animationIteration',
-  DOMTopLevelEventTypes.TOP_ANIMATION_START, 'animationStart',
-  DOMTopLevelEventTypes.TOP_CAN_PLAY, 'canPlay',
-  DOMTopLevelEventTypes.TOP_CAN_PLAY_THROUGH, 'canPlayThrough',
-  DOMTopLevelEventTypes.TOP_DURATION_CHANGE, 'durationChange',
-  DOMTopLevelEventTypes.TOP_EMPTIED, 'emptied',
-  DOMTopLevelEventTypes.TOP_ENCRYPTED, 'encrypted',
-  DOMTopLevelEventTypes.TOP_ENDED, 'ended',
-  DOMTopLevelEventTypes.TOP_ERROR, 'error',
-  DOMTopLevelEventTypes.TOP_GOT_POINTER_CAPTURE, 'gotPointerCapture',
-  DOMTopLevelEventTypes.TOP_LOAD, 'load',
-  DOMTopLevelEventTypes.TOP_LOADED_DATA, 'loadedData',
-  DOMTopLevelEventTypes.TOP_LOADED_METADATA, 'loadedMetadata',
-  DOMTopLevelEventTypes.TOP_LOAD_START, 'loadStart',
-  DOMTopLevelEventTypes.TOP_LOST_POINTER_CAPTURE, 'lostPointerCapture',
-  DOMTopLevelEventTypes.TOP_PLAYING, 'playing',
-  DOMTopLevelEventTypes.TOP_PROGRESS, 'progress',
-  DOMTopLevelEventTypes.TOP_SEEKING, 'seeking',
-  DOMTopLevelEventTypes.TOP_STALLED, 'stalled',
-  DOMTopLevelEventTypes.TOP_SUSPEND, 'suspend',
-  DOMTopLevelEventTypes.TOP_TIME_UPDATE, 'timeUpdate',
-  DOMTopLevelEventTypes.TOP_TRANSITION_END, 'transitionEnd',
-  DOMTopLevelEventTypes.TOP_WAITING, 'waiting',
+  'abort', 'abort',
+  ANIMATION_END, 'animationEnd',
```

Comment:
Are these intentionally not changed over?

Reply:
It’s calculated dynamically like before. Vendor prefixed. (I do wonder if we can drop that though.)